### ISSUE TITLE
The 'More actions' control cannot be accessed using voice commands such as Show number, Click, Focus, or Move to within the code editor pane.: A11y_VSCode_VoiceAccess

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#283001]
### GitHub Tags:
#A11yTCS; #A11yMAS;#Visual Studio Code Client;#BM_VisualStudioCodeClient_Win32_JULY2024; #Win32;#DesktopApp;#A11ySev3;#WCAG4.3.1;#No Disruption of Accessibility Features;#A11yeDAD;#VoiceAccess;

### Environment and OS details:
Version: 1.108.0-insider 
OS Build: 26200.7392

Pre-Requisites: 

1.Turn on Voice Access by Navigating to Settings> Accessibility> Speech> Voice Access or Search 'Voice Access' on Start. 

### Repro Steps:

1. Open VS Code.
2. Open new file, select Jupyter notebook and press enter.
3. Tab till more actions button in code cell.
4. Verify that 'More actions' control cannot be accessed using voice commands such as Show number, Click, Focus, or Move to within the code editor pane.

### Actual Result:
 'More actions' control is not accessible using voice access Show number/Click/Focus/Move to Commands present in code editor pane.

### Expected Result:
The 'More actions' control should be accessible using Voice Access commands, allowing users to interact with it by saying commands like Click 'More actions', Show number, Focus, or Move to.

### User Impact:
Users relying on Voice Access may struggle to interact with the 'More actions' control, limiting their ability to perform tasks efficiently.

### Attachment:

https://github.com/user-attachments/assets/74ed970d-17f5-43f6-be14-f2f68e228157

**Comment by AnKushSingh05** [comment]
> Thanks, does it work for the other actions in that toolbar? Like delete cell?

No, for other buttons also in the toolbar voice access is not working.
